tools/pot/openvino/tools/pot/algorithms/quantization/utils.py

```diff
@@ -18,7 +18,9 @@
                           'CPU': 'cpu.json',
                           'GNA': 'gna.json',
                           'GPU': 'cpu.json',
-                          'VPU': 'vpu.json'}
+                          'VPU': 'vpu.json',
+                          'CPU_SPR': 'cpu.json',
+                          'GPU_ATS': 'cpu.json'}
```

Comment:
Since we don't have a difference between SPR and ATS  for now and this is an experimental feature, let's introduce only one new target, e.g. `CPU_SPR`.